super-pane/search-field/select-field-to-search-for.tsx

```diff
@@ -0,0 +1,34 @@
+import { MenuButton, Button, Menu, MenuItem } from '@sanity/ui';
+import React from 'react';
+import FieldToSearchFor from './FieldToSearchFor';
+
+interface SelectFieldToSearchForProps {
+  currentField: FieldToSearchFor,
+  fieldsToChooseFrom: FieldToSearchFor[],
+  onFieldSelected: (field: FieldToSearchFor) => void
+}
+
+const SelectFieldToSearchFor: React.FC<SelectFieldToSearchForProps> = ({ currentField, onFieldSelected, fieldsToChooseFrom }) => {
```

Comment:
how would you feel about the name `SearchSelectField`?

Reply:
Absolutely, but at what variable should I replace? 🤠 line 8 or 11?